tools/vdebug/vdebug.cc

```diff
@@ -29,100 +29,203 @@ auto detector = std::make_shared<DepthImageObstacleDetector>(gzrs_camera);
 auto gz_quad = std::make_shared<MavQuadCopter>();
 std::vector<Obstacle> obstacles;
 
-static const int windowWidth = 1000;
-static const int windowHeight = 1000;
-static double eyeX = 0;
-static double eyeY = 0;
-static double eyeZ = 0;
+static int window;
+static int windowWidth = 1280;
+static int windowHeight = 720;
+static double eyeX = 40;
+static double eyeY = -30;
+static double eyeZ = 15;
+static double zoom = 20;
 
 static int enterX = 0, enterY = 0;
 
-void mouseMove(int mouseX, int mouseY)
+void draw_blade(float x, float y, float z)
 {
-    double angle = 3.14 * float(enterX + mouseX) / windowWidth;
-    eyeX = 100 * sin(angle);
-    eyeY = 100 * cos(angle);
-    eyeZ = (enterY + mouseY) / 10 ;
+    glPushMatrix();
 
-    glutPostRedisplay();
+	glColor3f(0.4f, 0.4f, 0.4f);
+
+    GLUquadricObj *cylinder;
+    cylinder = gluNewQuadric();
+
+    GLUquadricObj *circle;
+    circle = gluNewQuadric();
+
+    glTranslatef(x, y, z);
+    gluCylinder(cylinder, 0.3f, 0.3f, 0.05f, 32, 32);
+    gluDisk(circle, 0.0, 0.3f, 32, 1);
+    glTranslatef(0.0f, 0.0f, 0.05f);
+    gluDisk(circle, 0.0, 0.3f, 32, 1);
+
+    glPopMatrix();
 }
 
-void updateMouse(int button, int state, int mouseX, int mouseY)
+void draw_vehicle()
 {
-    if (button != GLUT_LEFT_BUTTON)
-        return;
-
-    if (state == GLUT_DOWN) {
-        enterX = enterX - mouseX;
-        enterY = enterY - mouseY;
-    } else if (state == GLUT_UP) {
-        enterX = enterX + mouseX;
-        enterY = enterY + mouseY;
-    }
+    glPushMatrix();
```

Comment:
yeah, it was tabs instead of spaces. It is weird since my editor will never use tabs. I probably copied a line from somewhere with a tab and it propagated. Anyway, I fixed it locally.